dds_web/web/user.py

```diff
@@ -604,6 +604,8 @@ def password_reset_completed():
             for unit_admin in unit_admins_to_contact:
                 dds_web.utils.send_project_access_reset_email(unit_admin, email[0], token)
 
+    action_logger = structlog.getLogger("actions")
+    action_logger.info(f"{flask.request.endpoint}", user=user.name, resource=flask.request.path)
```

Comment:
We definitely want to see the `username` in the log, but maybe having the name is also good, so something like 
```suggestion
    action_logger.info(f"{flask.request.endpoint}", user=user.name, username=user.username, resource=flask.request.path)
```

I was wondering about `flask.request.endpoint`, which in this case results in the same thing as the `flask.request.path`, but maybe it's good to have in case the URL path of the request is different from the view function handling the request

Reply:
I’ve done that now, thanks for the discussion!